docker-compose.apprise.yml

```diff
@@ -14,6 +14,11 @@ services:
       placement:
         constraints:
         - node.role == manager
+    environment:
```

Comment:
There already is an `environment` section above. Please move the new variables there as well.